### PR TITLE
[master] Zil/add more metrics

### PR DESCRIPTION
**Pull request by Steve-White-UK**
## Description
As the names suggests more an more metrics for the platform

## Backward Compatibility

- [ X] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [ ] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: src/libUtils/Metrics.cpp [hunk 1]
```diff
@@ -44,13 +48,87 @@ void Metrics::Init() {
...
+    std::unique_ptr<metrics_sdk::MeterSelector> observable_meter_selector{
+            new metrics_sdk::MeterSelector(name, version, schema)};
+    std::unique_ptr<metrics_sdk::View> observable_sum_view{
+            new metrics_sdk::View{name, "test_description", metrics_sdk::AggregationType::kSum}};
+    p->AddView(std::move(observable_instrument_selector), std::move(observable_meter_selector),
+               std::move(observable_sum_view));
+
+    // histogram view
+    std::string histogram_name = name + "_histogram";
+    std::unique_ptr<metrics_sdk::InstrumentSelector> histogram_instrument_selector{
```
Comment (bzawisto) [inline src/libUtils/Metrics.cpp:115]: Could it be implemented as std::make_unique<> ?

File: src/libData/AccountStore/AccountStore.cpp [hunk 2]
```diff
@@ -518,6 +521,8 @@ bool AccountStore::UpdateAccountsTemp(
   LOG_GENERAL(WARNING, "[AS] Finished Processing <"
                            << transaction.GetTranID() << "> ("
                            << (status ? "Successfully)" : "Failed)"));
+
+  AccountStoreSC::m_transactionLatency = ((double)r_timer_end(tpStart)/1000.0);
```
Comment (bzawisto) [inline src/libData/AccountStore/AccountStore.cpp:525]: better to use static_cast<>.

**Review by bzawisto — APPROVED** [review]

File: src/libData/AccountStore/AccountStore.cpp (hunk 2 above)
Comment (Steve-White-UK) [reply src/libData/AccountStore/AccountStore.cpp:525]: Yes, that's the old c programmer coming out, he is still in there somewhere, again this functionality is temporary as traces and spans carry the timing information, this is just to get a metric onto the dashboard this week. I will mark the code.

Comment (Steve-White-UK) [reply src/libData/AccountStore/AccountStore.cpp:525]: changed in code anyway.

**Pull request merged**